adafruit_bme680.py

```diff
@@ -381,14 +388,14 @@ def _read_calibration(self):
         self._heat_val = self._read_byte(0x00)
         self._sw_err = (self._read_byte(0x04) & 0xF0) / 16
 
-    def _read_byte(self, register):
+    def _read_byte(self, register: int) -> int:
         """Read a byte register value and return it"""
         return self._read(register, 1)[0]
 
-    def _read(self, register, length):
+    def _read(self, register: int, length: int) -> None:
```

Comment:
Since the intention of this method is to service as a blueprint for subclasses of `Adafruit_BME680` (IDEs will warn if this function isn't implemented), it'd be nice to blueprint what type it should return:

```suggestion
    def _read(self, register: int, length: int) -> bytearray:
```